packages/near-fast-auth-signer-e2e-tests/scripts/faucet.js

```diff
@@ -0,0 +1,87 @@
+const { CovalentClient } = require('@covalenthq/client-sdk');
+
+// Same addresses as the ones in constants.ts, they can't be imported here because this script is run by node.js
+const ETH_PERSONAL_KEY_ADDRESS = '0xf64750f13f75fb9e2f4d9fd98ab72d742d1e33eb';
+const BNB_DOMAIN_KEY_ADDRESS =  '0x81d205120a9f04d3f1ce733c5ed0a0bc66714c71';
+
+function fromWei(wei) {
+  return Number(BigInt(wei)) / 1e18;
+}
```

Comment:
there is exact same function exist on `ethers` on `near-fast-auth-signer/package.json` where:
```
const { ethers } = require('ethers');

function fromWei(wei) {
  return ethers.utils.formatEther(wei);
}
```
wonder if we should just import existing one?